internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/integration/utils/VeniceServerWrapper.java

```diff
@@ -394,6 +396,12 @@ public int getGrpcPort() {
     return serverProps.getInt(GRPC_READ_SERVER_PORT);
   }
 
+  public boolean isIsolatedIngestionEnabled() {
+    IngestionMode mode =
+        IngestionMode.valueOf(serverProps.getString(SERVER_INGESTION_MODE, IngestionMode.BUILT_IN.toString()));
```

Comment:
I think you're right. I've added a `throws` to `IngestionMode.valueOf()` to make it more obvious about the need for error handling and added a catch for a possible exception.

I'm going to just keep the return value as `false` by default unless specified otherwise in the supplied `serverProps`.